### PR TITLE
Return the package associated with a bz update.

### PR DESCRIPTION
**Pull request by ralphbean**

### CODE REVIEW

File: fedmsg_meta_fedora_infrastructure/bz.py [hunk 1]
```diff
@@ -145,6 +150,12 @@ def _gather_emails(self, msg):
 
         return users
 
+    def packages(self, msg, **config):
+        component = msg['msg']['bug']['component']
+        if component in bz_components_that_are_not_packages:
+            return set()
```
Comment (pypingou) [inline fedmsg_meta_fedora_infrastructure/bz.py]: Shouldn't we try to extract the package name then? Maybe from the title?

Comment (ralphbean) [inline fedmsg_meta_fedora_infrastructure/bz.py]: hotness has had a lot of trouble trying to do that :(

We could try.. but lots of people use different/non-standard formats when they type in the info.

Comment (pypingou) [inline fedmsg_meta_fedora_infrastructure/bz.py]: ok let's forget it then

**Comment by pypingou** [comment]
:+1: for me

**Comment by ralphbean** [comment]
Thanks @pypingou!

**Pull request merged**